### PR TITLE
Use account as submitting party

### PR DESCRIPTION
**Pull request by orelbn**
*Issue #:* /bcgov/entity#17061

*Description of changes:*
- Changes submitting party to be based on account info rather than manufacturer table for manufacturer registration

*Screenshots:*

Before:

image

After:

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts [hunk 1]
```diff
@@ -147,14 +146,7 @@ export const useMhrInformation = () => {
   }
 
   const parseSubmittingPartyInfo = (accountInfo: AccountInfoIF): void => {
-    const submittingParty = {
-      businessName: accountInfo.name,
-      address: accountInfo.mailingAddress,
-      emailAddress: accountInfo.accountAdmin.email,
-      email: accountInfo.accountAdmin.email,
```
Comment (orelbn) [inline ppr-ui/src/composables/mhrInformation/useMhrInformation.ts:154]: SubmittingPartyIF doesn't have the field `email`. So I removed it.

File: ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts [hunk 2]
```diff
@@ -139,7 +139,6 @@ export const useNewMhrRegistration = () => {
     const data = await getMhrManufacturerInfo()
     setMhrHomeDescription({ key: 'manufacturer', value: data.description.manufacturer })
     setMhrHomeDescription({ key: 'certificationOption', value: HomeCertificationOptions.CSA })
-    setMhrRegistrationSubmittingParty(data.submittingParty)
```
Comment (orelbn) [inline ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts:142]: Set before component is mounted.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1464-k381d0a9.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1464-k381d0a9.web.app

**Review by dimak1 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]
Looks great again.

**Review by cameron-eyds — APPROVED** [review]
Looks good!

**Pull request merged**